backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -110,6 +111,7 @@
 
   private static final Logger LOGGER =
       LogManager.getLogger(ExperimentRunDAORdbImpl.class.getName());
+  private static final boolean DO_OVERWRITE = false;
```

Comment:
```suggestion
  private static final boolean OVERWRITE_VERSION_MAP = false;
```

making it clear from name the usage of the variable

Reply:
renamed